### ISSUE TITLE
LocalFreshFood.io

### ISSUE THREAD

**Issue by scottvaladez** [district0x/district-proposals#58]
PURPOSE:
To create a network of local marketplaces to provide an efficient mechanism to distribute fresh organic home-grown and local organic nursery/farm grown food to local populations at decreased cost and with increased health benefits while bringing the societal benefit of significantly decreasing utilization of limited resources including carbon-based fuels, and water.

BACKGROUND:
There is significant waste of limited resources of oil/carbon-based fuel in the transportation and packaging of fresh food.  Furthermore transportation of fruits and vegetables requires utilization of pesticides and preservatives which have a negative impact on the quality of the food and the health of the consumers.  There is also significant waste in the food production and transportation process.

There has been a trend in society towards use of locally grown, fresh, organic foods. This includes use of water-conserving aquaponic and hydroponics techniques.  Aquaponic also has the potential to produce fish in a healthy, sustainable way, which takes pressures off of natural fisheries.

DESCRIPTION - COMMUNITY OPERATION:

KEY DEFINITIONS:
USERS - CONSUMERS AND PRODUCERS: Two key actors in the environment: consumers and producers.  Any individual could be one or both.  Consumers purchase food. Producers produce food. 

GEOGRAPHICAL DEFINITION OF CONSUMER'S MARKET: The local food marketplace would be defined by consumer-selected distance from any/all producers based.

PRODUCER INVENTORY: Producers will have an interface to reveal current inventory of upcoming (on plant, not yet fresh) food and currently ripe (ready to consume) food.

CONSUMER INTERFACE: Consumer interface would include search and sort functionality including distance, ripeness, farming process (traditional, organic, hydroponic, aquaponic), #available, etc.

Users could look at any producer to see their current inventory.

PRICE MECHANISM - FREE MARKET: Pricing would be based on auction-based system with reserve amount determined by farmer. There could be a buy-now price also. Sales could be monetized in terms of any number of cryptocurrencys/tokens based on acceptance by producer.  Alternatively District0x could interface with exchange services to provide the conversion process for transactions.

FOOD RESERVATION/USER REPUTATION: Food could be reserved based on request combined with a reputation rating which would be included for both consumers and producers (similar to Amazon.com).  Producers or consumers may set a reputation threshold in their search/sort interface or in their reserve optionality.

GOVERNANCE: Governance could be both local and general with the goal of improving the user experience, providing vital free market information to participants, and to promote the general success of the venture.

EFFICIENT DISTRIBUTION OF PRODUCERS BASED ON FREE MARKET FUNCTION: As an appendage, the community might provide means for individuals to invest in and organize tokenized small, local food production operations to meet market demands for specific undersupplied and underserved populations.

In summary, with the trends in healthier eating habits and localization of food production, the advancement in conservation friendly aquaponic and hydroponics farming techniques, and the advent of blockchain and specifically the innovative developing marketplaces of Distric0x, LocalFreshFood's time has come!

If you support this idea, please commit your tokens! Peace. :)

**Comment by AngeloAdam** [comment]
please send me a direct message on district slack regarding your proposal. @angelo

**Comment by scottvaladez** [comment]
Author slack username is scottvaladez. Thx.

_Sent from my Samsung SM-G955U using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Alexand0x** [comment]
How would the physical distribution of the food work?

**Comment by scottvaladez** [comment]
The idea is that there are a lot of local family, back-yard gardens, and hopefully more local greenhouses and small farms, such that you have a selection of very close producers (closer than your local grocery store).  So the producers are all located on a map based on your location, and you will just select producer or from their inventory, and perhaps reserve some tomatoes, etc.  Then you just walk or bike or drive to their location.

We would have a reputation function built in.  So feedback will accumulate on the various producers and consumers so transactions could be performed safely and efficiently to people's standards.

**Comment by KennyC37** [comment]
I really like this idea, as a chef, I love anything that employs the use of local farms/purveyors and try my best to use them even if it's at a higher cost. I believe that buying local not only promotes your own brand, but also builds the community around you, therefore making the whole area flourish a little more. Which should make the community stronger and more attractive for more development (not that more development is always a good thing).

Is this more focused on local retail or are you guys thinking of trying to get this going for wholesale also? Currently in Rhode island, our restaurant uses Farm Fresh mobile and Whatsgood Marketplace by Crave Foodservice. Both are great at connecting to local purveyors and farms to businesses. I'm more curious about what exactly the benefit will be for using a blockchain on this project opposed to the traditional route. More secure/more payment options?
